lib/rules/time-no-imperceptible/README.md

```diff
@@ -1,5 +1,7 @@
 # time-no-imperceptible
 
+***Deprecated: this rule is outside the scope of stylelint's functionality. See [the release planning docs](https://stylelint.io/user-guide/release-planning/) for details.***
```

Comment:
This `time-no-imperceptible` rule is being replaced with a more appropriately and more versatile rule named `time-min-milliseconds`. See #2249